config/views.yaml

```diff
@@ -113,10 +109,10 @@ component_readiness:
           - runc
           - crun
       compare_variants:
-        Topology:
-          - single
+        Architecture:
+          - arm64
```

Comment:
note that `multi-a-a` jobs count as arm64, but other heterogeneous jobs (e.g. `multi-x-ax`) do not.